package.json

```diff
@@ -56,9 +59,7 @@
     "webpack": "^2.7.0"
   },
   "peerDependencies": {
-    "material-ui": "^0.18.7",
-    "react": "^15.3.2",
```

Comment:
confused why react is no longer a peerDependency - isn't it required for the library to work?